viz_scripts/mapping_dictionaries.ipynb

```diff
@@ -13,73 +12,29 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "improved-tribute",
    "metadata": {},
    "outputs": [],
    "source": [
     "df_pur= pd.read_csv(r'auxiliary_files/purpose_labels.csv')\n",
     "df_re = pd.read_csv(r'auxiliary_files/mode_labels.csv')\n",
-    "df_ei = pd.read_csv(r'auxiliary_files/energy_intensity.csv')\n",
+    "df_EI = pd.read_csv(r'auxiliary_files/energy_intensity_updated.csv')\n",
```

Comment:
I don't see this file in the PR?

Reply:
My mistake. The name of the actual file should be the same as the previous one. I forgot to change it in the mapping jupyter notebook. I updated the mapping jupyter notebook to read the energy_intensity.csv